benchmarks/benchmarkMatrixVectorKernels.cpp

```diff
@@ -34,135 +34,165 @@ namespace benchmarking
   } \
   return
 
-#define MATRIX_VECTOR_KERNEL_RAJA( a_ij, b_j, c_i ) \
-  RAJA::forall< POLICY >( RAJA::TypedRangeSegment< INDEX_TYPE >( 0, N ), \
-                          [=] LVARRAY_HOST_DEVICE ( INDEX_TYPE const i ) \
+#define MATRIX_VECTOR_KERNEL_RAJA( N, M, a_ij, b_j, c_i ) \
+  forall< POLICY >( N, [=] LVARRAY_HOST_DEVICE ( INDEX_TYPE const i ) \
   { \
     for( INDEX_TYPE j = 0; j < M; ++j ) \
     { \
       c_i += a_ij * b_j; \
```

Comment:
```suggestion
      c_i = c_i + a_ij * b_j; \
```

also line 32 in this file.

Reply:
Done.